helm/vitess/templates/_vttablet.tpl

```diff
@@ -217,7 +217,7 @@ spec:
   image: "vitess/vttablet:{{$vitessTag}}"
   livenessProbe:
     httpGet:
-      path: /debug/vars
+      path: /debug/status
```

Comment:
We should be able to add `/debug/health` as a readinessProbe on vttablet if we set `podManagementPolicy: Parallel` on the StatefulSet. That makes sense anyway because vttablets don't need to be started in any particular order, so there's no need to wait.

Reply:
That's a good choice, but I don't think it solves the problem I encountered. I wasn't able to run `InitShardMaster` because it never reported as ready. I'll give it another try and report back whether or not it works.